src/user/models/kyc/kyc-webhook.service.ts

```diff
@@ -74,15 +75,24 @@ export class KycWebhookService {
             city: userData.location,
             zip: userData.zip,
             phone: userData.phone,
-            //TODO change for KYC Update v2
-            kycStatus: KycCompleted(userData.kycStatus) ? KycWebhookStatus.FULL : KycWebhookStatus.NA,
+            kycStatus:
+              KycCompleted(userData.kycStatus) && spiderData?.chatbotResult
+                ? KycWebhookStatus.FULL
+                : KycCompleted(userData.kycStatus)
+                ? KycWebhookStatus.LIGHT
+                : userData.kycStatus === KycStatus.REJECTED
+                ? KycWebhookStatus.REJECTED
+                : KycWebhookStatus.NA,
```

Comment:
Perhaps a separate method for this to restructure it a bit and make it more readable? E.g.
```ts
if (KycCompleted(...)) {
   return spiderData?.chatbotResult ? KycWebhookStatus.FULL : KycWebhookStatus.LIGHT;
} else if (...) {
   ...
} else {
   return KycWebhookStatus.NA;
}
```